zip-0207.rst

```diff
@@ -0,0 +1,277 @@
+less than its end height, but not less than its start height.
+
+Each funding stream has an associated set of recipient addresses. Each address is used for at most 17500
+blocks, creating a roughly-monthly sequence of funding periods. The address to be used for a given block
+height is defined as follows (using ``SlowStartShift`` as-defined in [#block-subsidy]_)::
+
+    AddressChangeInterval = 17500
+    FundingStream.AddressIndex(height) =
+        floor((height - FundingStream.StartHeight) / AddressChangeInterval)
+    Address(height) = FundingStream.Addresses[FundingStream.AddressIndex(height)]
+
+- To-do: adjust ``AddressChangeInterval``, taking #3672 and #3690 (both reducing block times) into account.
+
+Consensus rules
+---------------
+
+Prior to activation of the Blossom network upgrade, the existing consensus rule for payment of the original
+Founders' Reward is enforced. [#original-fr-consensus-rule]_
+
+Once the Blossom network upgrade activates:
+
+- The existing consensus rule [#original-fr-consensus-rule]_ is no longer active.
+- The coinbase transaction in each block MUST contain at least one output per active funding stream that pays
+  the stream's value to the stream's recipient address for the block's height.
+
+Stream definitions
+------------------
+
+The three logical funding streams described above each start at the Blossom activation height, and end at the
+first block reward halving. They are defined as follows:
+
+======== ================ ================== ================================
+ Stream  Value (zatoshis)    Start height              End height
+======== ================ ================== ================================
+   FR       184250000     Blossom activation SlowStartShift + HalvingInterval
+  ZF E       36000000     Blossom activation SlowStartShift + HalvingInterval
+ZECC SR      29750000     Blossom activation SlowStartShift + HalvingInterval
+======== ================ ================== ================================
+
+- To-do: specify the correct values.
+- To-do: specify the correct start height.
+- To-do: adjust the end height, taking #3672 and #3690 (both reducing block times) into account.
+
+The sets of recipient addresses are defined as follows:
+
+.. code:: cpp
+
+    FS_ADDRESSES_FR = [
+    ];
+
+    FS_ADDRESSES_ZF_E = [
+    ];
+
+    FS_ADDRESSES_ZECC_SR = [
+    ];
+
+- To-do: will the existing FR addresses be used for either the Zcash Company strategic reserve or the remainder, or will fresh addresses be used for all three?
+- To-do: specify the sets of FR addresses.
+- To-do: require that the three FR address sets are PGP-signed with appropriate keys.
+
+Example implementation
+----------------------
+
+.. code:: cpp
+
+    struct FundingPeriod {
+        CAmount value,
+        int startHeight,
+        int endHeight,
+    };
+
+    enum FundingStream {
+        FS_FR,
```

Comment:
Rename the constant to match the new name for "The ZECC Equity Fund" instead of repurposing "Founders' Reward".